src/component-library/Meter/Meter.tsx

```diff
@@ -79,8 +81,16 @@ const Meter = ({ value = 0, ranges, onChange, ...props }: MeterProps): JSX.Eleme
   }, [onChange, ranges, status, value]);
 
   return (
-    <StyledWrapper>
+    <StyledWrapper $showRanges={showRanges}>
       <StyledMeter $width={width} $hasIndicator={!!status} {...props} />
+      {showRanges && (
```

Comment:
Minor, but could this conditional be above StyledMeter to avoid using the fragment?

Reply:
Ah ignore that :) just realised that StyledMeter can be shown without ranges, so the answer is obviously no